lib/engine/game/g_18_uruguay/step/dividend.rb

```diff
@@ -41,6 +41,16 @@ def auto_actions(entity)
             [Engine::Action::Dividend.new(current_entity, kind: 'withhold')]
           end
 
+          def payoff_loans(entity)
+            loans_to_pay_off = [(entity.cash / 100).floor, entity&.loans&.size].min
```

Comment:
You don't need the safe navigation operator here. It doesn't do any harm, but you're testing `entity.loans.size` before invoking this method, so you're never going to have nil values.
```suggestion
            loans_to_pay_off = [(entity.cash / 100).floor, entity.loans.size].min
```